cmd/ssh_usercert_gen/auth_oauth2.go

```diff
@@ -0,0 +1,201 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"golang.org/x/net/context"
+	//"golang.org/x/oauth2"
+	"io/ioutil"
+	"log"
```

Comment:
Please consider using the logging support from the Dominator codebase and pass in the logger.